From 84385bcf6fa6e9cc29f466f70e667f5379dfc656 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Sun, 15 Sep 2019 07:34:33 +0530
Subject: [PATCH] gnulib-tests: undef MinGW's dummy pthread_sigmask define
 sooner

Fixes:

```
In file included from build\mingw\multilib\x86_64-w64-mingw32\sysroot\mingw\include\signal.h:10,
                 from ../gnulib-lib/signal.h:52,
                 from pthread_sigmask.c:20:
pthread_sigmask.c:34:1: error: expected identifier or '(' before numeric constant
 pthread_sigmask (int how, const sigset_t *new_mask, sigset_t *old_mask)
 ^~~~~~~~~~~~~~~
```
---
 gettext-tools/gnulib-tests/pthread_sigmask.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gettext-tools/gnulib-tests/pthread_sigmask.c b/gettext-tools/gnulib-tests/pthread_sigmask.c
index 3e40cfc..20034ef 100644
--- a/gettext-tools/gnulib-tests/pthread_sigmask.c
+++ b/gettext-tools/gnulib-tests/pthread_sigmask.c
@@ -30,9 +30,9 @@
 # include <unistd.h>
 #endif
 
+#undef pthread_sigmask
 int
 pthread_sigmask (int how, const sigset_t *new_mask, sigset_t *old_mask)
-#undef pthread_sigmask
 {
 #if HAVE_PTHREAD_SIGMASK
   int ret;
-- 
2.20.1.windows.1

